lms/templates/courseware/program_marketing.html

```diff
@@ -1,3 +1,424 @@
+              <div class="details">
+                <div class="name">
+                  <a href="#instructor-details-${index}" class="instructor-label">${instructor.get('name')}</a>
+                </div>
+                <div class="dept">${instructor.get('organization')}</div>
+              </div>
+
+              <div class="modal modal-custom" id="instructor-details-${index}">
+                <a href="#" class="focusKeeper"></a>
+                <div class="btn-close modal_close"><a href="#"><i class="fa fa-close"></i></a></div>
+                <div class="modal-body">
+                  <div class="modal-header">
+                    <div class="instructor-data">
+                      <div class="thumbnail">
+                        <img src="${instructor['image']}" alt="${instructor.get('name')}"/>
+                      </div>
+                      <h3>
+                        <span class="instructor-name">${instructor.get('name')}</span>
+                        <span>
+                          ${Text(_('{position} at {organization}')).format(
+                            position=instructor.get('title'),
+                            organization=instructor.get('organization')
+                          )}
+                        </span>
+                      </h3>
+                    </div>
+                  </div>
+                  <div class="instructor-bio">${instructor['bio']}</div>
+                </div>
+                <a href="#" class="focusKeeper"></a>
+              </div>
+            </div>
+          % endfor
+
+          <div class="pagination hidden-xs">
+            <div class="pages">
+              ## Translators: Pagination range contains ordinal numbers of the first and the last instructor
+              ## whose information is shown on the current page.
+              ${Text(_("{pagination_range} of {number_of_instructors}")).format(
+                pagination_range=HTML('<span class="pagination-start"></span>-<span class="pagination-end"></span>'),
+                number_of_instructors=HTML('<span class="instructor-size">{number_of_instructors}</span>').format(
+                  number_of_instructors=len(instructors)
+                )
+              )}
+            </div>
+            <div class="controls">
+              <a href="#" id="pagination-previous">
+                <span class="fa fa-chevron-up" aria-hidden="true"></span>
+                <span class="sr" aria-hidden="true">${_("Previous page")}</span>
+              </a>
+              <a href="#" id="pagination-next">
+                <span class="fa fa-chevron-down" aria-hidden="true"></span>
+                <span class="sr" aria-hidden="true">${_("Next page")}</span>
+              </a>
+            </div>
+          </div>
+        </div>
+        <blockquote class="program-quote">
+          <div class="quote">
+            % if endorsement:
+              <span class="pull-right hidden-sm">
+                % if endorser:
```

Comment:
@marjev We should check for endorsement_image here too. Its possible that endorser is not null but image is null.